scripts/tests/run_test_suite.py

```diff
@@ -232,7 +243,8 @@ def cmd_run(context, iterations, all_clusters_app, lock_app, ota_provider_app, o
         ota_provider_app=[ota_provider_app],
         ota_requestor_app=[ota_requestor_app],
         tv_app=[tv_app],
-        bridge_app=[bridge_app]
+        bridge_app=[bridge_app],
+        python_yaml_tester_cmd=['python3'] + [python_yaml_tester]
```

Comment:
```suggestion
        python_yaml_tester_cmd=['python3', str(python_yaml_tester)]
```

perhaps?

Reply:
That doesn't work, it causes an exec failure later on.